### DISCUSSION TITLE
Keep getting "0 Missing Albums Found"

### DISCUSSION THREAD

**Discussion by wjbeckett** [RandomNinjaAtk/docker-lidarr-extended#123] [Q&A] [answered]
I keep seeing the following in the logs:
```
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: Downloading page 1... (0 - 94 of 94 Results)
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: 0 MISSING ALBUMS FOUND
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: FINDING CUTOFF ALBUMS sorted by date
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: 0 CUTOFF ALBUMS FOUND
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: Searching for 0 items
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: No items to find, end
2022-11-10 07:07:00 :: Audio :: 1.0.274 :: Script end...
```
It seems like the script is finding 94 results in the missing items list, but does nothing with them.
They are valid albums and I can manually search Deezer and find them. 

Any Idea how I get around this and make the script search for those items?

**Comment by RandomNinjaAtk** [comment, marked as answer]
This happens because the script already looked for those items and did not find them. It prevents constantly searching for the same thing repeatedly over multiple script runs.

If you want to have it try again, you need to delete the contents of this folder: `/config/extended/logs/notfound`

The script will periodically, retry notfound albums to see if anything has changed, but its only like once every 30 days.

Edit:
If you add a new artist or new albums to your wanted list, it should still search for those... without the need to remove any files manually. 

What your seeing is by design, but maybe the logging output could be improved to make it more clear on what is happening...

Edit 2:
Another thing to keep in mind,  you will likely need to delete the folder contents of: `/config/extended/logs/downloaded` 
Because the script checks if it already downloaded an album previously, and will not re-download it based on log data in those folders.

**Reply by wjbeckett** [reply]
That's great. Thank you very much for the info. I'll try that.

**Reply by wjbeckett** [reply]
Okay, thanks. How can I look at why the grabs/matches aren't happening? I have wound it back to 50% match and I've still got a bunch of albums that aren't being grabbed.

**Reply by RandomNinjaAtk** [reply]
Logs is the only way, you should read this:

https://github.com/RandomNinjaAtk/docker-lidarr-extended/issues/53#issuecomment-1219955593
